ops-devnet/devnet-up.sh

```diff
@@ -97,6 +98,25 @@ fi
 L2OO_ADDRESS="0x6900000000000000000000000000000000000004"
 COLOSSEUM_ADDRESS="0x690000000000000000000000000000000000000D"
 VALPOOL_ADDRESS="0x6900000000000000000000000000000000000005"
+VALIDATOR_MNEMONIC="test test test test test test test test test test test junk"
+VALIDATOR_HD_PATH="m/44'/60'/0'/0/1"
+
+# Deposit into ValidatorPool to be a validator.
+(
+  cd "$KROMA_VALIDATOR"
+  echo "Deposit into ValidatorPool to be a validator..."
+  go run cmd/main.go \
+    --l1-eth-rpc $L1_URL \
+    --rollup-rpc "" \
+    --l2oo-address "" \
+    --colosseum-address "" \
+    --valpool-address $VALPOOL_ADDRESS \
+    --challenger.poll-interval 0s \
+    --mnemonic "$VALIDATOR_MNEMONIC" \
+    --hd-path $VALIDATOR_HD_PATH \
```

Comment:
Is there a way to exclude unnecessary flags when running balance subcommand?

Reply:
I cannot find any feature provided by `urfave/cli` to override or omit required flags from a parent command in a subcommand, so I guess the only way to omit unnecessary flags in a subcommand would be to replace the required flag with an optional one, but I don't think that's a good idea.